docs/dsp/parametriceq.md

```diff
@@ -0,0 +1,39 @@
+---
+title: DSP Parametric Equalizer
+description: Explanation of the use and effect of the DSP Parametric Equalizer filter
+---
```

Comment:
Can we add a small note on the page that this feature is available since 2.4.0 ?

Reply:
Done